Tests/NIOSSLTests/NIOSSLIntegrationTest.swift

```diff
@@ -1004,7 +1006,7 @@ class NIOSSLIntegrationTest: XCTestCase {
         let serverCtx = try configuredSSLContext()
         var clientConfig = TLSConfiguration.makeClientConfiguration()
         clientConfig.certificateVerification = .noHostnameVerification
-        clientConfig.trustRoots = .file("/tmp")
+        clientConfig.trustRoots = .file(tmpDir)
```

Comment:
I think this is the only one of the changes in this file we actually need.

Reply:
Yep, changed this pull to only change this line in this file.